<?xml version="1.0"?>
<launch>

  <arg name="CAM" default="3"/>
<!--  Available Dictionariytypes:
  ALL_DICTS,
  ARUCO, ARUCO_MIP_36h12, ARUCO_MIP_25h7, ARUCO_MIP_16h3,
  ARTAG, ARTOOLKITPLUS, ARTOOLKITPLUSBCH,
  TAG16h5, TAG25h7, TAG25h9, TAG36h11, TAG36h10, CHILITAGS-->

  <node type="aruco3_test" name="aruco3_test$(arg CAM)_2" pkg="calib" respawn="false" output="screen">
    <param name="topic_in_image" type="string" value="/genicam_cam$(arg CAM)/image_raw"/>
    <param name="marker_size" value="100"/>
    <param name="resize_factor" value="1"/>
    <param name="camera_parameter" value=""/>
    <param name="dictionary" value="TAG16h5"/>
    <param name="config_file_save" value="$(find calib)/cfg/aruco_config.yaml"/>
  </node>
</launch>
